um_canonicalize/tags.yaml

```diff
@@ -185,11 +211,14 @@ categories:
   'Information Structure':
     - TOP
     - FOC
+  New:
```

Comment:
Is this a typo? What type of feature class is `New`?

Reply:
I created it temporarily to add extra LGSPEC tags (where we provided feature names rather than numbers). E.g.,  Mordvinic languages contain many language-specific tags, and might be better to provide at least some name for it. E.g., LGSPEC_AMP stands for having an additive clitic that serves as an intensifier particle in Moksha/Erzya. See here: https://docs.google.com/spreadsheets/d/1DXXiJQlOKpgJ0oa50hAvAr6YdLDaIWKnOWQNA2TEDkY/edit?usp=sharing Can you suggest a possible solution here?